### DISCUSSION TITLE
模拟抽卡问题

### DISCUSSION THREAD

**Discussion by AinoAty** [Arondight/Adachi-BOT#820] [Q&A] [answered]
作者大大你好，请问怎么修改180上限，我看别人的使用有不同样式的 500以上的统计操作

**Comment by mark9804** [comment, marked as answer]
https://github.com/Arondight/Adachi-BOT/blob/6029cf4c29268954bf80b9e45701fe41d8b19b4c/src/plugins/gacha/index.js#L22

把那个 `180` 改成你自己想要的值

**Reply by AinoAty** [reply]
> https://github.com/Arondight/Adachi-BOT/blob/6029cf4c29268954bf80b9e45701fe41d8b19b4c/src/plugins/gacha/index.js#L22
> 
> 把那个 `180` 改成你自己想要的值

已经设置了，但是群内使用 抽卡 200  他绘图还是显示的180

**Reply by AinoAty** [reply]
> 

在群内使用help命令 貌似只生效了帮助文本上的数字变化

**Reply by AinoAty** [reply]
> > 
> 
> 在群内使用help命令 貌似只生效了帮助文本上的数字变化

请问是把command.yml 转移到congif文件夹内修改这个顺序么

**Reply by mark9804** [reply]
重启一下
